### PR TITLE
Add apptainer profile

### LINKED ISSUE #2244: Add `apptainer` container engine support
### Description of feature

Needs to be added in nextflow.config

Docs here: https://www.nextflow.io/docs/latest/config.html?highlight=apptainer#scope-singularity (Currently missing a header though...)

### PR DESCRIPTION
**Pull request by jfy133**
Add apptainer profile as now supported by NXF (I think added by @matthdsm into the main NXF repo?)

Closes #2244 

(More info: https://www.nextflow.io/docs/latest/container.html note website broken so apptainer section missing header, but will be fixed soon)

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Review by matthdsm — APPROVED** [review]
Should be okay I think!
PS: _requested_ by me, not added

**Comment by jfy133** [comment]
You were involved 

image

;)

File: nf_core/pipeline-template/.github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -45,6 +45,6 @@ body:
         * Nextflow version _(eg. 22.10.1)_
         * Hardware _(eg. HPC, Desktop, Cloud)_
         * Executor _(eg. slurm, local, awsbatch)_
-        * Container engine: _(e.g. Docker, Singularity, Conda, Podman, Shifter or Charliecloud)_
+        * Container engine: _(e.g. Docker, Singularity, Conda, Podman, Shifter, Charliecloud, or Apptainer)_
```
Comment (mashehu) [inline nf_core/pipeline-template/.github/ISSUE_TEMPLATE/bug_report.yml:48]:
```suggestion
        * Container engine: _(e.g. Docker, Singularity/Apptainer, Conda, Podman, Shifter or Charliecloud)_
```
Apptainer is just renamed singularity, no? I would therefor treat it in that way.

Comment (jfy133) [reply nf_core/pipeline-template/.github/ISSUE_TEMPLATE/bug_report.yml:48]: No not necessarily. They plan to make differences in the future IIRC

Comment (jfy133) [reply nf_core/pipeline-template/.github/ISSUE_TEMPLATE/bug_report.yml:48]: OK I 'm correct there, but I would rather just have it as a separate thing to follow the same scheme as they are separate binaries (and not confuse others)

**Review by mashehu — APPROVED** [review]

**Pull request merged**